### PR TITLE
Relay: upgrade kiwicom/relay and kiwicom/fetch

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: packages/relay/index.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 // @flow
 
 import * as React from 'react';
-import Relay from '@kiwicom/relay';
-import { type GraphQLTaggedNode as _GraphQLTaggedNode } from 'react-relay'; // eslint-disable-line
+import {
+  commitMutation as _commitMutation,
```
Comment (RobinCsl) [inline packages/relay/index.js]: Is it just to put emphasis on its _internal API_ nature?

File: app/core/src/screens/hotelsStack/placepicker/HotelCityQuery.js [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
-} from '@kiwicom/mobile-relay';
+import { PublicApiRenderer, graphql } from '@kiwicom/mobile-relay';
 
 import type { HotelCityQueryResponse as HotelCityType } from './__generated__/HotelCityQuery.graphql';
 import HotelCityList from './HotelCityList';
 
 type Props = {|
   +onPress: () => void,
-  +query: GraphQLTaggedNode,
+  +query: string,
```
Comment (RobinCsl) [inline app/core/src/screens/hotelsStack/placepicker/HotelCityQuery.js]: Why is this change necessary?

File: packages/relay/index.js (hunk 1 above)
Comment (tbergq) [reply packages/relay/index.js]: No, it is because I do export a const named commitMutation, so there would be a naming collision

File: app/core/src/screens/hotelsStack/placepicker/HotelCityQuery.js (hunk 2 above)
Comment (tbergq) [reply app/core/src/screens/hotelsStack/placepicker/HotelCityQuery.js]: Because the type of query is string, and not GraphQLTaggedNode.

Comment (tbergq) [reply app/core/src/screens/hotelsStack/placepicker/HotelCityQuery.js]: It is not like the query from graphql tagget template. It is a prop called query

File: packages/relay/index.js (hunk 1 above)
Comment (RobinCsl) [reply packages/relay/index.js]: Ah right. 😄 I should have expanded the file view!
